pkg/apis/apps/v1beta1/zz_generated.conversion.go

```diff
@@ -291,7 +290,8 @@ func autoConvert_v1beta1_DeploymentSpec_To_extensions_DeploymentSpec(in *v1beta1
 		return err
 	}
 	out.Selector = (*meta_v1.LabelSelector)(unsafe.Pointer(in.Selector))
-	if err := api_v1.Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(&in.Template, &out.Template, s); err != nil {
+	// TODO: Inefficient conversion - can we improve it?
```

Comment:
Saw that as well and now some other people. It does not happen all the time though. Not commiting those changes at least helps to get CI green. Possibly the Makefile.generated_files dependency logic is broken and we generate in the wrong order. Just a guess.